src/analysis/complex/basic.lean

```diff
@@ -0,0 +1,208 @@
+/-
+Copyright (c) Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel
+-/
+import analysis.calculus.deriv analysis.normed_space.finite_dimension
+
+/-!
+# Normed space structure on `ℂ`.
+
+This file gathers basic facts on complex numbers of an analytic nature.
+
+## Main results
+
+This file registers `ℂ` as a normed field, expresses basic properties of the norm, and gives
+tools on the real vector space structure of `ℂ`. Notably, in the namespace `complex`,
+it defines functions:
+
+* `re_linear_map`
```

Comment:
The mathematician in me still cries a bit when I see these really long names. Would it be evil to call these `linear_map.re`, so that one could `open complex.linear_map` to get the short name?

Reply:
I don't know if it is evil, but I have made the change.